### PR TITLE
Refactor to get product code from subscriber DB

### PR DESCRIPTION
**Pull request by zaro0508**
Instead of getting the product code from the SSM we get the
product code from the dynamo db which contains the
subscriber info.  Getting product code from DB is easier.

### CODE REVIEW

File: set_tags/utils.py [hunk 1]
```diff
@@ -251,25 +251,55 @@ def get_marketplace_customer_id(synapse_id):
 
   return customer_id
 
+def get_marketplace_product_code(synapse_id):
```
Comment (brucehoff) [inline set_tags/utils.py:254]: Why not combine this with get_marketplace_customer_id() so you get the customer Id and product code in a single database call?

File: set_tags/utils.py [hunk 2]
```diff
@@ -251,25 +251,55 @@ def get_marketplace_customer_id(synapse_id):
...
-
-      marketplace_customer_id = get_marketplace_customer_id(synapse_id)
-      if marketplace_customer_id:
-        tags.append({'Key': f'{MARKETPLACE_TAG_PREFIX}:customerId', 'Value': marketplace_customer_id})
+  marketplace_customer_id = get_marketplace_customer_id(synapse_id)
+  log.debug(f'marketplace customer ID: {marketplace_customer_id}')
+  if marketplace_customer_id:
+    tags.append({'Key': f'{MARKETPLACE_TAG_PREFIX}:customerId', 'Value': marketplace_customer_id})
+
+  marketplace_product_code = get_marketplace_product_code(synapse_id)
```
Comment (brucehoff) [inline set_tags/utils.py:299]: As mentioned above, you could have a method `get_marketplace_customer_info` that returns both the customer Id and product code in a single call.

**Review by brucehoff — CHANGES_REQUESTED** [review]
Just a small change:  It would simplify the code to combine the retrieval of customer Id and product code in a single database call.

File: set_tags/utils.py (hunk 1 above)
Comment (zaro0508) [reply set_tags/utils.py]: done

File: set_tags/utils.py (hunk 2 above)
Comment (zaro0508) [reply set_tags/utils.py]: done

**Review by brucehoff — APPROVED** [review]

**Pull request merged**